src/components/dialog-full-screen/dialog-full-screen.js

```diff
@@ -91,16 +103,44 @@ class DialogFullScreen extends Modal {
   }
 
   /**
-   * Returns title prop wrapped in <h2> if title is a string otherwise returns the title prop directly.
-   * Dialog main classes.
+   * Overrides the original function to disable the document's scroll.
+   */
+  get onOpening() {
+    Browser.addClass(this.document.body, DIALOG_OPEN_BODY_CLASS);
```

Comment:
When you use `elem.classList.add(name)` you don't need to check whether the class has already been added: if it has, it doesn't get added again.